modules/auxiliary/gather/cisco/cisco_rv320_config.rb

```diff
@@ -0,0 +1,117 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Auxiliary::Scanner
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'         => 'Cisco RV320/RV326 Configuration Disclosure',
+      'Description'  => %q{
+          A vulnerability in the web-based management interface of Cisco Small Business
+          RV320 and RV325 Dual Gigabit WAN VPN Routers could allow an unauthenticated,
+          remote attacker to retrieve sensitive information. The vulnerability is due
+          to improper access controls for URLs. An attacker could exploit this
+          vulnerability by connecting to an affected device via HTTP or HTTPS and
+          requesting specific URLs. A successful exploit could allow the attacker to
+          download the router configuration or detailed diagnostic information. Cisco
+          has released firmware updates that address this vulnerability.
+        },
+      'References'     =>
+        [
+          ['EDB', '46262'],
+          ['BID', '106732'],
+          ['CVE', '2019-1653'],
+          ['URL', 'https://seclists.org/fulldisclosure/2019/Jan/52'],
+          ['URL', 'https://bst.cloudapps.cisco.com/bugsearch/bug/CSCvg42801'],
+          ['URL', 'http://www.cisco.com/en/US/products/csa/cisco-sa-20110330-acs.html']
+        ],
+      'Author'         =>
+        [
+          'RedTeam Pentesting GmbH <release@redteam-pentesting.de>'
```

Comment:
Since this module was written by you, you should add yourself and note everyone's contributions.